text/0266-move-oteps-to-spec.md

```diff
@@ -0,0 +1,68 @@
+# Move OTEPS to the Specification repository
+
+Let's move OTEP documentation and PRs back into the github.com/open-telemetry/opentelemetry-specification repository.
+
+## Motivation
+
+Moving OTEPs back into the specification solves two main issues:
+
+- Maintaining its tooling infrastructure (currently woefully out of date)
+- Bringing it into the existing triage and voting process currently within the
+  Specification.
+
+## Explanation
+
+Originally, OTEPs were kept as a separate repository to keep disjoint/disruptive designs as a separate repository.  There are a few differences between a normal PR and an OTEP:
+
+- OTEPs are expected to be directional and subject to change when actually entered into the specification.
+- OTEPs require more approvals than specification PRs
+- OTEPs have different PR worklfows (whether due to accidental omission or conscious decision), e.g. staleness checks, linting.
+
+As OpenTelemetry is stabilizing, the need for OTEPs to live outside the specification is growing less, and we face challenges like:
+
+- Keeping OTEP tooling up to date
+- Advertising the repositories existence
+  - New contributors to OpenTelemetry often can't find recorded decision that exist in OTEPs.
+  - Getting reviews from folks used to checking the Specification repository, but not the less-frequently-worked-on OTEP repository.
+
+To solve these, let's move OTEPs into a directory within the [specification repository](github.com/open-telemetry/opentelemetry-specification).
+We would also update all tooling and expected reviews to match existing standards for OTEPs.  Given the maintainers of OTEPs are the same as
+maintainers of the specification, this should not change the bar for acceptance.
+
+## Internal details
+
+The following changes would occur:
+
+- The following files would be moved to the specification repo:
+  - `text/` directory -> `oteps/text/`
+  - `0000-template.md` -> `oteps/0000-template.md`
+- Update the specification `Makefile` to include linting, spell checking, link checking and TOC-ing the oteps directory.
+- A one-time cleanup of OTEP markdown upon import to the specification repository.
+- Close existing OTEP PRs and ask folks to reopen against the specification repository.
```

Comment:
What about issues in the OTEP repo? Should those be closed too like PRs?

Reply:
I guess they are going to be moved to https://github.com/open-telemetry/opentelemetry-specification

Reply:
I missed these, but yes - I think those move to the spec repo.  I think all of them should have been OTEPs, and it's likely we close and/or ask folks to open formal OTEPs for these.

They current fall below the radar of triage and we need to address that.